build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
```

Comment:
This is making a copy of `teamcity-nightly-roachtest.sh` and editing it, right? I find that sort of thing fairly sad in general. I assume there are reasons, but then please outline them and add comments here and in the other copy that make it very clear that any change there also needs to be made here and vice versa.

Reply:
I refactored out a big chunk of the script to avoid the duplication.